preon-binding/src/main/java/org/codehaus/preon/codec/ObjectCodecFactory.java

```diff
@@ -188,6 +196,11 @@ private AnnotatedElement hideChoices(AnnotatedElement metadata)
         return new HidingAnnotatedElement(BoundObject.class, metadata);
     }
 
+    // TODO: Remove this. It's in to prevent breaking of current usage of Preon.
```

Comment:
Do we need to leave in this todo comment?  This seems like a reasonable behavior to keep going forward, with field orders being optional

Reply:
Removed